packages/taquito-local-forging/src/michelson/codec.ts

```diff
@@ -97,6 +97,21 @@ export const extractRequiredLen = (value: Uint8ArrayConsumer, bytesLength = 4) =
   return value.consume(valueLen);
 };
 
+export const extractRequiredLenArrayItems = (
```

Comment:
can you give a brief explanation of what this method does? I think we can work on giving it another name.

Reply:
our extractRequiredLen takes in bytes that are prefixed with a length and return the value after 
this one takes in a string that contains multiple values prefixed with a length of the string of the array then the length of each item of the array